### ISSUE TITLE
Sammendrag mangler

### ISSUE THREAD

**Issue by sda030** [NIFU-NO/nifutypst#34]
abstract er definert i scope, men ikke implementert.
Sammendraget er vel unummerert, så kan være greit å ha et tydelig sted å plassere det, og måte å definere sammendraget.
Det kan bli litt begrensende å måtte skrive det i en YAML-oppføring for en qmd-fil, selv om det er enkleste måte å implementere det på (slik som forordet). Det er kanskje bedre for brukeren om det skrives i en div:
```quarto
::: {.abstract}
Her kan man skrive akkurat slik man er vant til i Quarto.
:::
```

**Comment by henrikkarlstrom** [comment]
Ja, jeg tok det ut av YAML-headeren. Sikkert lurt å gjøre det til en del av eksempel-malen, men det er nok noe brukeren selv må huske å spesifisere. Forordet må nesten være som et eget variabel-felt et sted, for det ligger midt inni front matter og ikke etter.
